src/windows_registry.rs

```diff
@@ -488,11 +488,24 @@ mod impl_ {
             .join("bin")
             .join(&format!("Host{}", host))
             .join(&host.to_lowercase());
-        let lib_path = path.join("lib").join(&target);
+        let lib_path = path
+            .join(if use_spectre_mitigated_libs() {
+                r"lib\spectre"
+            } else {
+                "lib"
+            })
+            .join(&target);
         let include_path = path.join("include");
         Some((bin_path, host_dylib_path, lib_path, include_path))
     }
 
+    fn use_spectre_mitigated_libs() -> bool {
+        match env::var("VSCMD_ARG_VCVARS_SPECTRE") {
```

Comment:
Could simplify to `env::var("VSCMD_ARG_VCVARS_SPECTRE") == Ok("spectre")`

Reply:
cc msrv is bumped to 1.53 , so you could use that api now